packages/core/src/types.ts

```diff
@@ -121,6 +121,8 @@ export interface EditorOptions {
   onFocus: (props: EditorEvents['focus']) => void;
   onBlur: (props: EditorEvents['blur']) => void;
   onDestroy: (props: EditorEvents['destroy']) => void;
+  onPaste?: (e: ClipboardEvent) => void
+  onDrop?: (e: DragEvent) => void
```

Comment:
I think these types are intentionally not optional, there is a Partial<EditorOptions> wherever we use it